editor/import/resource_importer_scene.cpp

```diff
@@ -1682,7 +1682,12 @@ void ResourceImporterScene::_generate_meshes(Node *p_node, const Dictionary &p_m
 				}
 
 				if (generate_lods) {
-					src_mesh_node->get_mesh()->generate_lods(merge_angle, split_angle);
+					// Calculate global scale not including the root scale
+					Vector3 global_scale = src_mesh_node->get_scale();
```

Comment:
Is it possible to use get_global_transform() * root.get_transform().affine_inverse().origin so this fix uses the transform caching? Order of operations on origin calculation doesn't matter.

Reply:
I recently woke up. 

I meant (src_mesh_node->get_global_transform() * root.get_transform().affine_inverse()).get_scale().

Reply:
`src_mesh_node->get_tree()` return null so should I just use a loop to get the root node?

Reply:
Yes. It was late.